cluster/charts/crossplane-controllers/templates/package-manager-deployment.yaml

```diff
@@ -72,6 +72,9 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        # The container name
+        - name: CONTAINER_NAME
+          value: {{ .Chart.Name }}
```

Comment:
Do you think it'd cause a problem if we used a hard-coded value as container name, like `manager` or sth, both here and `name` field of the container? I couldn't really get why `{{ .Chart.Name }} ` had been chosen.

Reply:
In the general case no, but if someone was running in host aware mode then they may have multiple manager deployments in the same name namespace but running against different API servers. If the manager deployment name was hardcoded then they would have collisions.

Reply:
> If the manager deployment name was hardcoded then they would have collisions.

That's not the deployment name though, it's the container's identifier in the `Pod`. Also, `.Chart.Name` is actually same for all Crossplane installations as opposed to `.Release.Name`.

Reply:
@muvaf good call! Yeah this shouldn't end up conflicting if hard-coded so that may be a better solution 👍